modxcms/revolution
# Changed old class name to FCQN in the class_key for resources in the tree

…when building the create resource URL.

### What does it do?
When creating a new resource from the plus button on the resource tree, the create URL was being built incorrectly.
This PR updates it to use `MODX\Revolution\modResource` instead of just `modResource`.

### Why is it needed?
The incorrect class name (or lexicon value) was showing in the class type field when creating a new resource via the plus button.

### How to test
Try creating a new resource via the plus button on the resource tree. I assume this will need the minified JS file to be updated, so you might not see the fix without that.

### Related issue(s)/PR(s)
Fix #15423
